bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/timeacct/TimeAcctAward.java

```diff
@@ -0,0 +1,82 @@
+package edu.gemini.spModel.timeacct;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.util.Objects;
+
+
+/**
+ * A tuple combining program and partner time awards.
+ */
+public final class TimeAcctAward implements Serializable {
```

Comment:
Not really, I guess.  It's siblings `TimeAcctAllocation` and `TimeAcctCategory`, and most of the client code that uses it as well, were written pre-Scala so it just seemed natural to do it in Java as well.